src/css/base.css

```diff
@@ -72,6 +72,11 @@ code {
   color: var(--color-brand-gray1);
 }
 
+a code {
+  color: var(--color-link);
+  background-color: transparent;
```

Comment:
I did check this for e.g. code listings with links, possibly locally (I think there is an example in Server docs I'll try to dig up).

That wasn't a problem because, as in your snippet above, it's `code a` whereas the CSS only applies to `a code` 👍

Reply:
Yeah what I'm thinking of is probably an edge case. This PR fixes the immediate problem. LGTM